closure/compiler/closure_js_library.bzl

```diff
@@ -74,8 +83,9 @@ closure_js_library = rule(
         "deps": JS_DEPS_ATTR,
         "exports": JS_DEPS_ATTR,
         "language": attr.string(default=JS_LANGUAGE_DEFAULT),
+        "suppress": attr.string_list(),
```

Comment:
Can you also update `README.md` to document the `suppress` option of ` closure_js_library`?